### PR TITLE
3614 im notifications modal community

### LINKED ISSUE #3614: notification modal on communities home
As a community member, I'd like to be able to access a list of community notifications from the community's home page.

- "moderation log" link under "Leadership" section should bring up a modal
- Make API request to notifications endpoint for the specific community
- Reuse account NotificationsList rendering (refactor if needed)

### PR DESCRIPTION
**Pull request by gl2748**
Closes #3614 

Screen Shot 2019-12-09 at 4 28 06 PM

TODO:

- [x] Add moderation log link to `CommunityPaneMobile.jsx`
- [x] Make sure pagination works for the NotificationList.jsx component when showing notifications inside a modal.

### CODE REVIEW

**Comment by roadscape** [comment]
@gl2748 could you please take a look at the merge conflict and ensure this feature works on `bridge-api-dev`? I resolved the conflict locally but receive a completely blank modal even though I see the results coming in successfully. Likely a conflicting change from another PR.

**Comment by gl2748** [comment]
@roadscape I've resolved the merge conflict.
We might want to change the copy from 'moderation log' to something different as the notifications list is displaying community activity such as subscriptions etc...
Screen Shot 2020-01-27 at 10 40 38 AM

File: src/app/components/cards/NotificationsList.jsx [hunk 1]
```diff
@@ -212,15 +209,7 @@ class NotificationsList extends React.Component {
                         </Callout>
                     )}
 
-                {loading && (
```
Comment (roadscape) [inline src/app/components/cards/NotificationsList.jsx]: Why remove the loading indicator? Would be nice to keep it  to provide feedback on poor connections.

**Comment by gl2748** [comment]
Added pagination to notifications dialog.

Screen Shot 2020-01-31 at 8 33 50 AM

File: src/app/components/cards/NotificationsList.jsx (hunk 1 above)
Comment (gl2748) [reply src/app/components/cards/NotificationsList.jsx]: This wasn't actually doing anything - vestige of some other loading state. For this component loading state is represented by the `notificationActionPending` prop.

Comment (roadscape) [reply src/app/components/cards/NotificationsList.jsx]: Hmm.. it was working before, but it's not on `bridge-api-dev`, so it must have been a regression in another PR

**Pull request merged**